src/main/java/com/wiley/driver/factory/capabilities/ChromeCaps.java

```diff
@@ -28,6 +30,11 @@ public ChromeOptions get() {
         ChromeOptions caps = getChromeOptions();
         if (!this.customCaps.asMap().isEmpty()) {
             caps.merge(this.customCaps);
+
+            // This is a workaround for ChromeOptions.merge() issue when passed arguments are not being assigned properly
+            Map chromeCapability = (Map) caps.getCapability(ChromeOptions.CAPABILITY);
```

Comment:
I suppose I could define ths map as TreeMap , butI'm not sure that forcing certain K,V types is a good idea in this case, since get() requires and returns generic/Object types, so I'll have to use unchecked typecasting instead of allowing the app to assign types during execution. What do you think?